doc/tutorial/printing_drawing.txt

```diff
@@ -14,6 +14,15 @@ compilation.  These two functions print expression graphs in different ways:
 Theano also provides :func:`pydotprint` that creates a *png* image of the function.
 You can read about them in :ref:`libdoc_printing`.
 
+.. note::
```

Comment:
done. thanks

On Wed, Jan 21, 2015 at 9:41 AM, Jesse Livezey <notifications@github.com>
wrote:

> In doc/tutorial/printing_drawing.txt
> <https://github.com/Theano/Theano/pull/2425#discussion_r23318013>:
>
> > @@ -14,6 +14,15 @@ compilation.  These two functions print expression graphs in different ways:
> >  Theano also provides :func:`pydotprint` that creates a *png* image of the function.
> >  You can read about them in :ref:`libdoc_printing`.
> >
> > +.. note::
>
> Actually, there could be another small change here that I missed. (added
> "will")
>
> .. note::
>
>     When printing Theano functions, they can sometimes be hard to
>     read.  To help with this, you can disable some Theano optimizations
>     by using the Theano flag:
>     ``optimizer_excluding=fusion:inplace``. Do not use this during
>     real job execution, as this will make the graph slower and use more
>     memory.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2425/files#r23318013>.
>